src/compiler/php_generator.cc

```diff
@@ -21,6 +21,7 @@
 #include "src/compiler/config.h"
 #include "src/compiler/generator_helpers.h"
 #include "src/compiler/php_generator_helpers.h"
+#include <google/protobuf/compiler/php/php_generator.h>
```

Comment:
I am not sure it is better to leave it here(only this file use it), or move it to config.h to make it looks like grpc::protobuf::compiler::php